src/test/java/macrobase/pipeline/BasicBatchedPipelineTest.java

```diff
@@ -18,8 +16,8 @@
 
 import static org.junit.Assert.*;
 
-public class BatchAnalyzerTest {
-    private static final Logger log = LoggerFactory.getLogger(BatchAnalyzerTest.class);
+public class BasicBatchedPipelineTest {
```

Comment:
There's some commenting already, but we could do better: https://github.com/stanford-futuredata/macrobase/blob/newDesign/src/test/java/macrobase/pipeline/BasicBatchedPipelineTest.java#L23